From 1cd48a21dce5e9edd6da530e8610b107dae53efb Mon Sep 17 00:00:00 2001
From: Nikita Tokarchuk <nikita@tokarch.uk>
Date: Thu, 28 Nov 2019 18:22:04 +0100
Subject: [PATCH 1/1] fix c11 threads

---
 configure.librdkafka | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.librdkafka b/configure.librdkafka
index 4c7d14a1..1d6135fb 100644
--- a/configure.librdkafka
+++ b/configure.librdkafka
@@ -43,7 +43,7 @@ function checks {
 
 
     # Use internal tinycthread if C11 threads not avaialable
-    mkl_lib_check "c11threads" WITH_C11THREADS disable CC "" \
+    mkl_lib_check "c11threads" WITH_C11THREADS disable CC "-lpthread" \
                   "
 #include <threads.h>
 
-- 
2.23.0

